pages/consortium.html

```diff
@@ -12,14 +12,37 @@
     hand we wish to make sure that the <strong>most pressing needs of the user community are addressed
     efficiently</strong> by providing a fantastic working environment to competent developers.</p>
 
+<p>The Coq Consortium is hosted by the Inria Foundation, a <strong>non-profit
+organization</strong> associated to Inria providing engineering resources for
```

Comment:
I don't believe emphasis is needed here, but do as you wish.

Reply:
It shouldn't be needed indeed, but the whole thing is a bit hard to read (font size too small, text too compact), so I'd rather keep the emphasis until we improve the website theme.